### PR TITLE
[LibFix] Fix `Container.run` library

### PR DESCRIPTION
**Pull request by vamahaja**
Fix consists of -
  1. Add missing parameter for `detach_key`.
  2. Fix library definition.
  3. Fix `image` and `rm` parameter condition

### CODE REVIEW

**Review by adityaramteke — APPROVED** [review]

**Comment by adityaramteke** [comment]
LGTM

File: cli/utilities/containers.py [hunk 1]
```diff
@@ -54,25 +54,27 @@ def __init__(self, nodes, package="podman"):
...
             volume (list): List of volumes
             ports (list): List of ports
-            rm (str): Image name to remove from background
-            detach (str): Image name to detach
             restart (str): Restart flag
+            detach_key (list): List of detach operation
+            detach (str): Image name to detach
             cmds (str): Other commands to be executed
         """
         if not image or not rm:
```
Comment (adityaramteke) [inline cli/utilities/containers.py:80]:
```suggestion
        if not image and not rm:
```
The condition will not be fulfilled when passing only the image.

**Review by adityaramteke — CHANGES_REQUESTED** [review]

File: cli/utilities/containers.py (hunk 1 above)
Comment (vamahaja) [reply cli/utilities/containers.py]: Fixed now

**Review by adityaramteke — APPROVED** [review]

**Review by MohitBis — APPROVED** [review]
LGTM

**Pull request merged**